test/uk/gov/hmrc/pillar2submissionapi/controllers/BTNSubmissionControllerSpec.scala

```diff
@@ -59,24 +59,18 @@ class BTNSubmissionControllerSpec extends ControllerBaseSpec {
     }
 
     "submitBTN() called with an invalid request" should {
-      "return InvalidJson response" in {
-
-        result(invalidRequestJson_data) shouldFailWith InvalidJson
-      }
+      "return InvalidJson response" in
+        result(invalidRequestJson_data).shouldFailWith(InvalidJson)
     }
 
     "submitBTN called with an invalid json request" should {
-      "return InvalidJson response" in {
-
-        result(invalidRequest_Json) shouldFailWith InvalidJson
-      }
```

Comment:
Was this change scalafmt or scala 3 related? Interesting that something seems to prefer the postfix style over braces. It seems to prefer being very explicit with braces and method calls in most cases.